### DISCUSSION TITLE
🦊 - Version 9.0.1.20 Beta ready for testing (Firefox)

### DISCUSSION THREAD

**Discussion by paulrouget** [aclap-dev/video-downloadhelper#559] [Announcements]
Download here: https://www.downloadhelper.net/beta

(Firefox only for now)

- updated locales
- remember last download location
- bigger fonts when selecting "Make addon popup larger"

**Comment by GrampaWildWilly** [comment]
I don't see any change from what 9.0.1.13a was giving me.

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/f7bcaf35-ddd2-4a45-ae58-4fa86541feee)

But that's not a complaint.  That looks perfectly fine to me.  Except for -- here he goes playing that same old tune again -- the all caps, bold, italic video title.

As for remembering the last download location, that does appear to be fixed.  The value of the Download Directory in the settings isn't changing.  I suppose that doesn't particularly matter.  That setting used to change with each download.  OK, so it works differently now.  I can deal with that.  It was far more important to remember the most recent download directory & that is now happening.

**Reply by GrampaWildWilly** [reply]
Come to think of it, that setting for the Download Directory seems pretty useless.  Once you do your first download, VDH resets to that directory as the default.  That changes when you download into some other directory.  The setting doesn't seem to serve any purpose now.  How much convincing do you need?

**Reply by GrampaWildWilly** [reply]
More come to think of it.  If a user has never used VDH before, you should make his first download a Download As.  That will force him to come face to face with the file selection dialog.  He will have to either accept what is proposed, presumably something in the User directory tree, or set what he really wants.  After that first execution, there will be a directory from a previous download that VDH can use from that point on.

And for those who must have political correctness, assume I mean "he or she/him or her" above in all the relevant spots.

**Reply by paulrouget** [reply]
> I don't see any change from what 9.0.1.13a was giving me.

The font is supposed to be much bigger (50% bigger) if you use the "Make popup larger" option. You don't see that?

> Come to think of it, that setting for the Download Directory seems pretty useless. Once you do your first download, VDH resets to that directory as the default.

Can you explain? That's not supposed to happen. That is the download directory for the regular "Download" action.

> That changes when you download into some other directory. The setting doesn't seem to serve any purpose now. How much convincing do you need?

I don't understand what you're saying. The expected logic is:

Default download directory is used for "Download" action and the first "Download As" action.
The next "Download As" will re-use the directory selected in the previous "Download As".

If that's not the behavior you see, let me know.
If that's not the behavior you want, explain to me what you expect.

**Reply by GrampaWildWilly** [reply]
Oh.  OK.  I've always had the wider menu so I didn't notice a change in the font size.  I've just experimented a bit with flipping that setting off & then on again & yes, the font does seem to be somewhat bigger with the wider menu.

Your explanation of how the download directory works is what I observe.  I always use Download As so the Download Directory setting is moot for me.  Since I don't use simple Download, I wasn't observing all the possibilities.  The thing I was expecting was the behavior from past releases.  Download As . . . well, to use the old terminology, Download (not Quick Download) would cause the setting for Download Directory to change value.  The descendant of Download is now Download As, which does not change the value of the setting.  The descendant of Quick Download is Download.  Download uses the setting.  Download As does not use the setting except for the very first time.  I had not grasped all the subtleties.  I've got it now.  Everything is fine.

**Comment by devilsandro** [comment]
> Baixe aqui: [ https://www.downloadhelper.net/beta ](https://www.downloadhelper.net/beta)
> 
> (Firefox apenas por enquanto)
> 
>     * localidades atualizadas
> 
>     * lembre-se do último local de download
> 
>     * fontes maiores ao selecionar "Aumentar o pop-up do complemento"

Good evening, I bought the key with the promise of it being for life, now in this new upgrade you hear a downgrade it only recognizes videos on YouTube up to 720 and the license is recognized but it keeps asking for an update to buy another key for life, maybe something will be able to download 1080 videos by purchasing it again that's right?

**Reply by paulrouget** [reply]
Please share the information about the version you're using. In the settings, click on the Copy button and paste it here.

**Reply by devilsandro** [reply]
Version 9.0.2.3a1 (beta), target mozilla, locale pt-BR. voltou a funcionar com essa versão obrigado por responder

**Comment by noddy33** [comment]
I love video downloadhelper - until the latest version. It was so brilliant. I'm on firefox and now it won't download anything. Any file I try I get O and when I have tried to delete and download an earlier version any file just shows queued. Such a shame.

**Reply by GrampaWildWilly** [reply]
If you open a new problem report using the guided process you can find via the Wiki (link to which appears at the top of every page in this forum), you will find that somebody will probably be able to help you.  But you have to open a separate problem report.  You must read & follow the instructions you will find during the guided problem creation process.  Do not skip any steps.  Every bit of the information you provide is important.  It will all be useful in problem diagnosis.

**Comment by noddy33** [comment]
Thanks!

**Comment by Maccadoodle67** [comment]
How do i stick to using Version 8.2.2.8 coz the New version keeps overriding it and I have to keep reinstalling  ???

**Comment by TEOL85** [comment]
I have to use old mode in it if I want to receive Mixcloud because in new mode it only shows bits and not a full 2 or 3 hour file like old mode does and old mode also says 192 and 64 whilst the new mode doesn't show anything. That is something with the old mode workaround required to get a full podcast but I hope it can be fixed in the new mode as well or I won't be using it.

Works well though.

It is annoying when I get the WEBM YouTube it defaults to 44100 128 sound when the master is 128-160 48000 on Vorbis sound and 128-160 44100 on the M4A especially when my other software YouTube Audio and Video Downloader broke last week and only gets html files now what we get in this better all in 1 software is lesser sound because it's not 48000 160 just before that other software broke though 320 48000 Vorbis came into the picture then 2 weeks before that software failed back to everything 160 48000 on Vorbis but it's miles ahead of 128 44100 vorbis it converts it to on download here but VDH is all I have so I hope to see proper 160 48000 Vorbis soon with even that 320 if it still exists on some tracks but went on the fritz before the other add on went on the friz.

But VDH was my lifeline to continue with my show so it did it's job just getting my things ready but I do need 48000 Vorbis pure re-joining without the converting changing any of the webm format to lower formats especially the sound on the highest video quality detected really.

VDH is brillent it is the most reliable as far as always working it's just the conversion side is a bit dodgy it doesn't do the purest conversion of video native and audio native on video and audio it converts it to it's own formats at the moment and totally wipes over the better khz 48000 with 44100 on 160-320 WEBMs ok when the highest picture and sound for it is 128 160 44100 m4a it isn't a drop at all but it is when you got a webm with the sound merging into it the 128-160 44100 m4a and not the native to the webm version 160-320 48000 Vorbis.

Well anyway it's working for me I just still get annoyed at fake sound quality on webm versions. It's detecting the webm video fine but it doesn't detect the vorbis dash audio which actaully belongs to that type of file and uses a replacement audio from the m4a dash which is fake to that version whichthe one converting currently is the dash audio for the mp4 not the webm at all.

Dash Audio for Vorbis needs to be put in the WebM not the m4a and then it would be correct with 48000 conversion.

Good work though Paul I just have an issue with the wrong dash sound with the wrong dash video merging into each other and what is weirder is these WEBM versions are detected as the wrong extension too as the actaul file for this version is webm.

Sometimes m4a has the better bitrate and quality when both are done in 128 because the vbr is sometimes higher on the m4a in that situration but generally the Vorbis is better as when it's not 128 it is always verying 160-320 formats with 48000 khz and should be the dash on the webm but then there is the quirk that it's not ven displaying as a webm on the tool neither when that is the name format of the higher compression audio and videos MKV is incorrect to what the native extension is.

This version is called .WEBM on the picture when downloading the dash natively but the sound is also callled .WEBM when downloading it seperate natively but has the name of Vorbis and joint together the format should be called .WEBM and the sound in mediainfo if the software gets the dashes fixed on the format should say a format between 128-160-192-256-320 and the khz should be 48000 if the dash version is joining and rejoining the correct format.

MKV extension and 44100 m4a sound at 128-145 VBR is completely incorrect for the webm though. M4A dash is 100% correct extension correct dashes joining in the merge though so it's good when the bitrate of Vorbis is 128 to go for that as the VBR on 128 44100 m4a is higher bitrate hits which is less of a drop than losing 48000 as audo tracks are often 44100 converted to 48000 on the Vorbis anyway so when Vorbis is 128 m4a is better but the 160-320 48000 Vorbis qualitys are the best on YouTube and storm over 128 44100 on both.

We just need it to correctly rejoin as a webm and use webm audio and webm video dash files for the correct joining on the better one.

MBPS detection video on MKV is probably the correct native HD WebM video already but incorrect extension as it is called WEBM natively and maybe should detect and download it as as otherwise it's confusing to the user although we probably know MKV is the same but labbeled differently anyway but m4a is the wrong dash sound for that file as the correct dash sound is also a webm but has 48000 khz show on the Media Info software when downloading it from my broken app before and doesn't tend to tell the bitrate only the khz anyway although when it's less or the same as 128 m4a in mbs of audio only download that is a 128 Vorbis and when it's the same as m4a in size it's better quality due to the 48000khz and if lower filesize it's a lower 128 VBR than the m4a so the mp4 version is better to download as  more bitrate is better than khz in that case.

But to match up which ones I need to get on VDH I need the WEBM to get all the correct WEBM dashes merging. 

Things might get even more complecated when you get into trash 96k webm and lower audio needs to be put on the trash 140 & 240 ect. formats on webm video to be authentic but 360 and above tend to use the good webm dash audio but VDH is a good tool as why go authentic at lower res when you can put the higher dash audio on WEBM and MP4 sides for everything.

Anyway keep up the great work Paul I hope we get that webm issue sorted I want my 48000 audio back after losing it because my other tool is broken I used to use the other tool for YouTube and VDH for podcast downloads and other video sites but now it's a lifeline I gotta use VDH for everything so need my correct 48000 webm dash audio files and dash WEBM video saving as a WEBM not a incorrect MKV format sooner rather than later but I still get decent sound from it anyway I'm not bothered but I still want it fixed merging the correct dashes and saving with the correct extension and the correct khz.
